recipes/nsight-compute/meta.yaml

```
@@ -0,0 +1,204 @@
+  - '$RPATH/libCommonProtoStreamSections.so'
+  - '$RPATH/libCore.so'
+  - '$RPATH/libCudaDrvApiWrapper.so'
+  - '$RPATH/libcuda.so.1'
+  - '$RPATH/libDevicePropertyProto.so'
+  - '$RPATH/libDeviceProperty.so'
+  - '$RPATH/libexporter.so'
+  - '$RPATH/libGenericHierarchy.so'
+  - '$RPATH/libGpuInfo.so'
+  - '$RPATH/libHostCommon.so'
+  - '$RPATH/libibmad.so.5'
+  - '$RPATH/libInterceptorInjectionTarget.so'
+  - '$RPATH/libInterfaceData.so'
+  - '$RPATH/libInterfaceSharedBase.so'
+  - '$RPATH/libInterfaceSharedCore.so'
+  - '$RPATH/libInterfaceSharedLoggers.so'
+  - '$RPATH/libInterfaceShared.so'
+  - '$RPATH/libLinuxPerf.so'
+  - '$RPATH/libNsysServerProto.so'
+  - '$RPATH/libNvtxExtData.so'
+  - '$RPATH/libProcessLauncher.so'
+  - '$RPATH/libStreamSections.so'
+  - '$RPATH/libSymbolAnalyzerLight.so'
+  - '$RPATH/libSymbolDemangler.so'
+  - '$RPATH/libTimelineAssert.so'
+  - '$RPATH/libTimelineUIUtils.so'
+  - '$RPATH/libTimelineWidget.so'
+  - '$RPATH/libucm.so.0'
+  - '$RPATH/libucp.so.0'
+  - '$RPATH/libucs.so.0'
+  - '$RPATH/libuct.so.0'
+  - '$RPATH/libicuuc.so.71'
+  - '$RPATH/libicudata.so.71'
+  - '$RPATH/libcurl.so.4'
+#not sure
+  - '$RPATH/libEGL.so.1'
+  - '$RPATH/libGL.so.1'
+  - '$RPATH/libjpeg.so.62'
+  - '$RPATH/libProtobufCommClient.so'
+  - '$RPATH/libProtobufCommProto.so'
+  - '$RPATH/libProtobufComm.so'
+  - '$RPATH/libtiff.so.5'
+  - '$RPATH/libSshClient.so' #not libssh
+#cdt-builds?
+  - '$RPATH/libibumad.so.3'
+  - '$RPATH/libibverbs.so.1'
+  - '$RPATH/libxshmfence.so.1'
+#qt-feedstock only has libQt5
+  - '$RPATH/libQt6WaylandClient.so.6'
+  - '$RPATH/libQt6Core.so.6'
+  - '$RPATH/libQt6Gui.so.6'
+  - '$RPATH/libQt6Widgets.so.6'
+  - '$RPATH/libNvQtGui.so'
+  - '$RPATH/libQt6Charts.so.6'
+  - '$RPATH/libQt6Concurrent.so.6'
+  - '$RPATH/libQt6DBus.so.6'
+  - '$RPATH/libQt6Network.so.6'
+  - '$RPATH/libQt6OpenGL.so.6'
+  - '$RPATH/libQt6OpenGLWidgets.so.6'
+  - '$RPATH/libQt6Positioning.so.6'
+  - '$RPATH/libQt6PrintSupport.so.6'
+  - '$RPATH/libQt6QmlModels.so.6'
+  - '$RPATH/libQt6Qml.so.6'
+  - '$RPATH/libQt6Quick.so.6'
+  - '$RPATH/libQt6StateMachine.so.6'
+  - '$RPATH/libQt6Svg.so.6'
+  - '$RPATH/libQt6UiTools.so.6'
+  - '$RPATH/libQt6WaylandEglClientHwIntegration.so.6'
+  - '$RPATH/libQt6WebChannel.so.6'
+  - '$RPATH/libQt6WebEngineCore.so.6'
+  - '$RPATH/libQt6WebEngineWidgets.so.6'
+  - '$RPATH/libQt6WlShellIntegration.so.6'
+  - '$RPATH/libQt6XcbQpa.so.6'
+  - '$RPATH/libQt6Xml.so.6'
+  - '$RPATH/libQtPropertyBrowser.so'
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - sysroot_{{ target_platform }} 2.17  # [linux]
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+  host:
+    - cuda-version {{ cuda_version }}
+    - boost-cpp
+    #- icu #not helping?
+    - dbus
+    - wayland  # [linux]
+    - libxkbcommon
+    - libexpat
+    - freetype
+    - fontconfig
+    - alsa-lib
+    - libglib
+    - libxcb
+    - libzlib
+    - nspr
+    - nss
+    - xorg-libxrender
```

This was later renamed

```suggestion
    - libxrender
```